.changeset/warm-bees-tie.md

```diff
@@ -0,0 +1,20 @@
+---
+'@sap-ux/axios-extension': patch
+'@sap-ux/backend-proxy-middleware': patch
+'@sap-ux/btp-utils': patch
+'@sap-ux/environment-check': patch
+'@sap-ux/fe-fpm-writer': patch
```

Comment:
Do we need to patch them? The `fe-fpm-writer` e.g. has only a change in the linting config, so it shouldn't affect the generated module.

Reply:
Yeah should no longer be needed.